ref-exchange/src/utils.rs

```diff
@@ -9,13 +9,13 @@ use uint::construct_uint;
 
 /// Attach no deposit.
 pub const NO_DEPOSIT: u128 = 0;
-/// hotfix_insuffient_gas_for_mft_resolve_transfer, increase from 5T to 20T
-pub const GAS_FOR_RESOLVE_TRANSFER: Gas = 20_000_000_000_000;
+/// hotfix_insuffient_gas_for_mft_resolve_transfer, increase from 20T to 25T for AS token contract.
+pub const GAS_FOR_RESOLVE_TRANSFER: Gas = 25_000_000_000_000;
 
 pub const GAS_FOR_FT_TRANSFER_CALL: Gas = 25_000_000_000_000 + GAS_FOR_RESOLVE_TRANSFER;
```

Comment:
May need `+10TGas` here to cover `GAS_FOR_FT_TRANSFER` cost. Or better change this to something like
```suggestion
pub const GAS_FOR_FT_TRANSFER_CALL: Gas = 20_000_000_000_000 + GAS_FOR_FT_TRANSFER + GAS_FOR_RESOLVE_TRANSFER;
```

Need to see where is this called

Reply:
> May need `+10TGas` here to cover `GAS_FOR_FT_TRANSFER` cost. Or better change this to something like
> 
> Need to see where is this called

In REF, `GAS_FOR_FT_TRANSFER_CALL` is used in implement of `mft_transfer_call` to deduce the gas transfered to the receiver of mft_on_transfer, in the form of `env::prepaid_gas() - GAS_FOR_FT_TRANSFER_CALL`.  

So, it will be better to using 25T to cover usage of the function itself and GAS_FOR_RESOLVE_TRANSFER for the self callback `mft_resolve_transfer`.  All remaining prepaid gas would go to the receiver side, where it may need that `GAS_FOR_FT_TRANSFER`. So, I think we don't need to add `GAS_FOR_FT_TRANSFER` into `GAS_FOR_FT_TRANSFER_CALL`.